bin/razor-admin

```diff
@@ -78,7 +86,7 @@ end
 
 # It is very important that razor/initialize isn't loaded until after
 # RACK_ENV has been set
-require_relative '../lib/razor/initialize'
+require 'razor/initialize'
```

Comment:
When referencing a file from a symlink, the relative path was being evaluated from the symlink location rather than the source location, which meant the libraries couldn't be found. Rather than loading a relative library, it was easier to add the razor libs directory to the Ruby load path. That, and @mckern said something about `required_relative` being deprecated